src/run.js

```diff
@@ -73,10 +74,25 @@ const loadConfig = testPath =>
 
 // Run tests on a given file
 const run = ({ testPath, config, globalConfig }) => {
-  const schema = require(testPath)
+  const { transform } = config
+
+  let schema
+  let fileContent = fs.readFileSync(testPath, { encoding: 'utf-8' })
+
+  if (
+    !config.transformIgnorePatterns
+      .map(x => new RegExp(x))
+      .find(x => x.test(testPath))
+  ) {
+    const transformer = transform.find(x => new RegExp(x[0]).test(testPath))
```

Comment:
`transform` is [not an array in the config](https://jestjs.io/docs/en/configuration#transform-objectstring-pathtotransformer--pathtotransformer-object); is this transformed by jest before the runner is called?  It looks like it might have been based on your use of `x[0]` here for the key and `x[1]` below for creating the transform object; if not you might want to use `Object.entries(transform).find()` instead.